.travis.yml

```diff
@@ -26,7 +26,7 @@ before_install:
 install:
   - conda create -q -n testenv --yes python=$TRAVIS_PYTHON_VERSION ipython pyzmq numpy scipy nose matplotlib pandas Cython patsy statsmodels flake8 scikit-learn seaborn runipy pytables networkx pandas-datareader matplotlib-tests joblib
   - source activate testenv
-  - pip install nose_parameterized contextlib2 logbook==0.10.1
+  - pip install nose_parameterized contextlib2 logbook==0.10.1 qrisk>=0.1.3
```

Comment:
Why does this differ from setup.py?  Instead of specifying it twice, could we pip install pyfolio itself here, which will look at any dependencies?  I notice that logbook and contextlib2 are not install requirements - why do we install them here?